csp/impl/pandas_perspective.py

```diff
@@ -12,6 +13,13 @@
 
 try:
     import perspective
+
+    if version.parse(perspective.__version__) >= version.parse("3"):
```

Comment:
We duplicate this version parsing logic in perspective.py, dataframe.py and pandas_perspective.py. Can we just have a single `perspective_common.py` file and then all 3 just import `_PERSPECTIVE_3` from it? I think it will be less error prone when we remove this logic down the road.

Reply:
thats probably out of scope for this PR since we're planning on moving to https://github.com/point72/csp-adapter-perspective fairly imminently

Reply:
Regardless of if we move away from the native perspective adapter in CSP, we shouldn't duplicate the same version checking code in 3 places.

Reply:
This is done.